app/javascript/pages/account/create-account.tsx

```diff
@@ -163,8 +163,8 @@ const onSubmit = (setError: (name: string, error: ErrorOption) => void) => (data
   }
   createAccount(userData, contactData)
     .then(() => {
-      // TODO: Redirect to Sign In page with modal message
-      console.log("Created an account.")
+      window.sessionStorage.setItem("redirect", "true")
+      window.location.replace("/sign-in")
```

Comment:
question: This is sending me to the Angular page (I'm testing the review app), what do I need to do to get redirected to the React page?